src/main/properties/datasets.properties

```diff
@@ -358,160 +427,219 @@ org.aksw.gerbil.datasets.definition.KORE50_Crunchbase.constructorArgs=${org.aksw
 ### Masakha
 org.aksw.gerbil.datasets.masakhaDatasetConfiguration.datasetdir=${org.aksw.gerbil.DataPath}/datasets/masakha
 ### Amharic
+org.aksw.gerbil.datasets.definition.Masakha-Amharic-Dev.group=MasakhaNER Amharic
```

Comment:
This should only be `MasakhaNER`. The same holds for the other datasets with this name.

Reply:
Done